test/integration/manifests/cilium/v1.14/cilium-agent/templates/daemonset.yaml

```diff
@@ -42,12 +42,6 @@ spec:
                 operator: In
```

Comment:
I think there should be dualstack daemonset for v1.14 too ?

Reply:
good catch, updated!